### PR TITLE
support mpp proto and exchanger operators

### PR DESCRIPTION
**Pull request by hanfei1991**

### CODE REVIEW

File: proto/executor.proto [hunk 1]
```diff
@@ -22,6 +22,8 @@ enum ExecType {
 	TypeLimit = 5;
 	TypeStreamAgg = 6;
 	TypeJoin = 7;
+	TypeExchanger = 8;
+	TypeExchangerClient = 9;
```
Comment (fzhedu) [inline proto/executor.proto:26]: rename to exchangeSender and ExchangeReceiver ?

Comment (hanfei1991) [reply proto/executor.proto:26]: ok

File: proto/executor.proto [hunk 2]
```diff
@@ -33,9 +35,30 @@ message Executor {
 	optional Aggregation aggregation = 5;
 	optional TopN topN = 6;
 	optional Limit limit = 7;
-	optional Aggregation stream_agg= 8;
+	optional ExchangeReceiver exchange_receiver = 8;
```
Comment (qw4990) [inline proto/executor.proto:38]: Why overwrite `stream_agg`?

Comment (hanfei1991) [reply proto/executor.proto:39]: Because it's unnessary and never used. We'd better remove the garbage as soon as we can.

**Comment by fzhedu** [comment]
LGTM

**Review by fzhedu — APPROVED** [review]
LGTM

**Review by qw4990 — APPROVED** [review]
LGTM

**Pull request merged**